lib/eco/treemanager.py

```diff
@@ -127,6 +128,8 @@ def right(self):
 
     def jump_left(self):
         self.node = self.find_previous_visible(self.node)
+        if self.node.symbol.name == "\r":
+            self.line -= 1
```

Comment:
That's because I wrote a commit on this branch, and you committed the same code on master in a separate commit. When we squash this PR I will remove the duplicate commit.